Junit/src/IPL.java

```diff
@@ -0,0 +1,364 @@
+import java.util.*;
```

Comment:
Do not create all required classes in the same file. 
segregate your code into meaningful files and packages.